services/keywords/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Use the official Python image.
+# https://hub.docker.com/_/python
+FROM python:3.7-slim
+
+# Copy local code to the container image.
+ENV APP_HOME /app
+WORKDIR $APP_HOME
+COPY . .
+
+# Install production dependencies.
+RUN pip install functions-framework
```

Comment:
This is unnecessary since `functions-framework` is already specified inside the `requirements.txt`
Having an extra `run` command generates an extra docker layer too.

Reply:
Huh yeah, that seems reasonable. I might have missed it when reading it, but this is from the [example dockerfile](https://github.com/GoogleCloudPlatform/functions-framework-python/blob/master/examples/cloud_run_http/Dockerfile), and within the same paragraph it tells you to put it in the requirements.txt https://github.com/GoogleCloudPlatform/functions-framework-python#cloud-runcloud-run-on-gke

Reply:
I'm going to leave the conversation open if anyone has an idea on why the functions framework left both

Reply:
I'm guessing that they did it that way so that there's a layer with that Python dependency and they "save" time when they rebuild the image with a changed `requirements.txt`. But in our case that makes no difference as our Cloud Build will always rebuild the Docker Image with each trigger.